src/pages/accessibility.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement } from 'react';
+import { TwoThirdsLayout } from '../layout/Layout';
+
+const title = 'Accessibility - Fares Data Build Tool';
+const description = 'Accessibility page for the Fares Data Build Tool';
+
+const Accessibility = (): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description}>
+            <h1 className="govuk-heading-l">Accessibility statement for Fares Data Build Tool</h1>
+            <p className="govuk-body">
+                This service is part of the wider GOV.UK website. There’s a separate accessibility statement for the
+                main GOV.UK website.
+            </p>
+            <p className="govuk-body">
+                This page only contains information about the Fares Data Build Tool service, available at
+                https://fares-data.dft.gov.uk
```

Comment:
Should this be an actual link or are we happy with it being just plain text?

Reply:
Could also do with a fullstop on the end of here to be consistent with the rest of the page?